<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>gestions clients</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="2" column="0">
           <widget class="QTabWidget" name="tabWidget_2">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab_3">
             <attribute name="title">
              <string>ajouter client</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QGroupBox" name="groupBox">
                  <property name="title">
                   <string/>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_6">
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_5">
                     <item>
                      <widget class="QGroupBox" name="groupBox_2">
                       <property name="title">
                        <string/>
                       </property>
                       <layout class="QHBoxLayout" name="horizontalLayout_18">
                        <item>
                         <widget class="QGroupBox" name="groupBox_3">
                          <property name="title">
                           <string>partenaire 1</string>
                          </property>
                          <layout class="QVBoxLayout" name="verticalLayout_7">
                           <item>
                            <layout class="QFormLayout" name="formLayout">
                             <item row="0" column="0">
                              <widget class="QLabel" name="cINPartenaire1Label">
                               <property name="text">
                                <string>CIN partenaire 1</string>
                               </property>
                              </widget>
                             </item>
                             <item row="0" column="1">
                              <widget class="QLineEdit" name="LineEditp1"/>
                             </item>
                             <item row="1" column="0">
                              <widget class="QLabel" name="prNomPartenaire1Label">
                               <property name="text">
                                <string>nom partenaire 1</string>
                               </property>
                              </widget>
                             </item>
                             <item row="1" column="1">
                              <widget class="QLineEdit" name="prNomPartenaire1LineEdit"/>
                             </item>
                             <item row="2" column="0">
                              <widget class="QLabel" name="nomPartenaire1Label">
                               <property name="text">
                                <string>prénom partenaire 1</string>
                               </property>
                              </widget>
                             </item>
                             <item row="3" column="0">
                              <widget class="QLabel" name="dateDeNaissancePartenaire1Label">
                               <property name="text">
                                <string>date de naissance partenaire 1</string>
                               </property>
                              </widget>
                             </item>
                             <item row="4" column="0">
                              <widget class="QLabel" name="tLPhonePartenaire1Label">
                               <property name="text">
                                <string>téléphone partenaire 1</string>
                               </property>
                              </widget>
                             </item>
                             <item row="4" column="1">
                              <widget class="QLineEdit" name="tLPhonePartenaire1LineEdit"/>
                             </item>
                             <item row="5" column="0">
                              <widget class="QLabel" name="mailPartenaire1Label">
                               <property name="text">
                                <string>mail partenaire 1</string>
                               </property>
                              </widget>
                             </item>
                             <item row="5" column="1">
                              <widget class="QLineEdit" name="mailPartenaire1LineEdit"/>
                             </item>
                             <item row="2" column="1">
                              <widget class="QLineEdit" name="nomPartenaire1LineEdit"/>
                             </item>
                             <item row="3" column="1">
                              <widget class="QDateEdit" name="dateEdit_2"/>
                             </item>
                            </layout>
                           </item>
                          </layout>
                         </widget>
                        </item>
                        <item>
                         <widget class="QGroupBox" name="groupBox_4">
                          <property name="title">
                           <string>partenaire 2</string>
                          </property>
                          <layout class="QVBoxLayout" name="verticalLayout_8">
                           <item>
                            <layout class="QFormLayout" name="formLayout_3">
                             <item row="0" column="0">
                              <widget class="QLabel" name="cINPartenaire1Label_2">
                               <property name="text">
                                <string>CIN partenaire 2</string>
                               </property>
                              </widget>
                             </item>
                             <item row="0" column="1">
                              <widget class="QLineEdit" name="cINPartenaire1LineEdit_2"/>
                             </item>
                             <item row="1" column="0">
                              <widget class="QLabel" name="nomPartenaire2Label">
                               <property name="text">
                                <string>nom partenaire 2</string>
                               </property>
                              </widget>
                             </item>
                             <item row="1" column="1">
                              <widget class="QLineEdit" name="nomPartenaire2LineEdit"/>
                             </item>
                             <item row="2" column="0">
                              <widget class="QLabel" name="prNomPartenaire2Label">
                               <property name="text">
                                <string>prénom partenaire 2</string>
                               </property>
                              </widget>
                             </item>
                             <item row="2" column="1">
                              <widget class="QLineEdit" name="prNomPartenaire2LineEdit"/>
                             </item>
                             <item row="3" column="0">
                              <widget class="QLabel" name="dateDeNaissancePartenaire2Label">
                               <property name="text">
                                <string>date de naissance partenaire 2</string>
                               </property>
                              </widget>
                             </item>
                             <item row="4" column="0">
                              <widget class="QLabel" name="tLPhonePartenaire2Label">
                               <property name="text">
                                <string>téléphone partenaire 2</string>
                               </property>
                              </widget>
                             </item>
                             <item row="4" column="1">
                              <widget class="QLineEdit" name="tLPhonePartenaire2LineEdit"/>
                             </item>
                             <item row="5" column="0">
                              <widget class="QLabel" name="mailPartenaire2Label">
                               <property name="text">
                                <string>mail partenaire 2</string>
                               </property>
                              </widget>
                             </item>
                             <item row="5" column="1">
                              <widget class="QLineEdit" name="mailPartenaire2LineEdit"/>
                             </item>
                             <item row="3" column="1">
                              <widget class="QDateEdit" name="dateEdit_3"/>
                             </item>
                            </layout>
                           </item>
                          </layout>
                         </widget>
                        </item>
                       </layout>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QFormLayout" name="formLayout_5">
                     <item row="0" column="0">
                      <widget class="QLabel" name="budgetLabel">
                       <property name="text">
                        <string>budget</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QLineEdit" name="budgetLineEdit"/>
                     </item>
                     <item row="1" column="0">
                      <widget class="QLabel" name="prFRenceLabel">
                       <property name="text">
                        <string>préférence</string>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="1">
                      <widget class="QLineEdit" name="prFRenceLineEdit"/>
                     </item>
                     <item row="2" column="1">
                      <widget class="QPushButton" name="pushButton">
                       <property name="text">
                        <string>ajouter</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_4">
             <attribute name="title">
              <string>afficher clients</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QTableView" name="tableView"/>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_5">
                <property name="title">
                 <string/>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QPushButton" name="pushButton_3">
                   <property name="text">
                    <string>supprimer client</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit"/>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_4">
                <property name="text">
                 <string>modifier client</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_2">
                <property name="text">
                 <string>trier client</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_6">
                <property name="text">
                 <string>rechercher client</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_5">
                <property name="text">
                 <string>envoyer mail</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>gestions réservations</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <widget class="QTabWidget" name="tabWidget_3">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_6">
           <attribute name="title">
            <string>ajouter réservation</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_13">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_12">
              <item>
               <layout class="QFormLayout" name="formLayout_7">
                <item row="0" column="0">
                 <widget class="QLabel" name="prixLabel">
                  <property name="text">
                   <string>prix</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="prixLineEdit"/>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="dateLabel">
                  <property name="text">
                   <string>date</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="typeLabel">
                  <property name="text">
                   <string>type</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QLineEdit" name="typeLineEdit"/>
                </item>
                <item row="6" column="0">
                 <widget class="QLabel" name="nombreInvitSLabel">
                  <property name="text">
                   <string>nombre invités</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="1">
                 <widget class="QLineEdit" name="nombreInvitSLineEdit"/>
                </item>
                <item row="8" column="0">
                 <widget class="QLabel" name="localisationLabel">
                  <property name="text">
                   <string>localisation</string>
                  </property>
                 </widget>
                </item>
                <item row="8" column="1">
                 <widget class="QLineEdit" name="localisationLineEdit"/>
                </item>
                <item row="10" column="1">
                 <widget class="QPushButton" name="pushButton_7">
                  <property name="maximumSize">
                   <size>
                    <width>500</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>ajouter</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="3" column="1">
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="5" column="1">
                 <spacer name="verticalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="7" column="1">
                 <spacer name="verticalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="9" column="1">
                 <spacer name="verticalSpacer_5">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="2" column="1">
                 <widget class="QDateEdit" name="dateEdit"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_7">
           <attribute name="title">
            <string>afficher réservations</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_15">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_14">
              <item>
               <widget class="QTableView" name="tableView_2"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_8">
              <property name="text">
               <string>supprimer réservation</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_9">
              <property name="text">
               <string>modifier réservation</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_10">
              <property name="text">
               <string>rechercher réservation</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_11">
              <property name="text">
               <string>trier réservations</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>bilan statistique</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
